### PR TITLE
Feature/buy life

### PR DESCRIPTION
**Pull request by KyleWong2510**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Could not previously buy lives

## What is the new behavior?

- Can now spend points to buy a new life
- Modal pops up to confirm the purchase
- Tests

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/BuyLifeModal/BuyLifeModal.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './BuyLifeModal.css'
+
+const BuyLifeModal = (props) => {
```
Comment (Billwilke42) [inline src/BuyLifeModal/BuyLifeModal.js:4]: Great job on this feature!!

File: src/BuyLifeModal/BuyLifeModal.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    const yesBtn = getByRole('button', { name: 'Yes' })
+    const noBtn = getByRole('button', { name: 'No' })
+
+    expect(title).toBeInTheDocument()
+    expect(text).toBeInTheDocument()
+    expect(yesBtn).toBeInTheDocument()
+    expect(noBtn).toBeInTheDocument()
+  })
+
+  it('should run a fn on click of the yes btn', () => {
```
Comment (Billwilke42) [inline src/BuyLifeModal/BuyLifeModal.test.js:38]: Nice work on the tests

File: src/Chalkboard/Chalkboard.js [hunk 3]
```diff
@@ -82,12 +85,22 @@ class Chalkboard extends React.Component {
...
+  toggleBuyLifeModal = () => {
+    this.setState({ isBuyingLives: !this.state.isBuyingLives })
+  }
+
+  buyLife = () => {
+    this.props.incrementLives()
+    this.props.decrementScore(300)
+    this.toggleBuyLifeModal()
+  }
+
```
Comment (lilianaweimer) [inline src/Chalkboard/Chalkboard.js:103]: Great idea to add this!

File: src/actions/index.test.js [hunk 4]
```diff
@@ -157,6 +157,17 @@ describe('actions', () => {
     expect(result).toEqual(expectedAction)
   })
 
+  it('Should increment the lives', () => {
```
Comment (Billwilke42) [inline src/actions/index.test.js:160]: Thanks for adding the action test

**Pull request merged**